### PR TITLE
Daman master1

### PR DESCRIPTION
**Pull request by damanveen**
pull requesting

### CODE REVIEW

**Comment by cmoihub** [comment]
StudentController should points to page

**Comment by cmoihub** [comment]
Possibly in Main Controller when you point to a page, you could move it to the appropriate controller for that page

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,6 @@ jdk:
 deploy:
   provider: heroku
   app:
-    noReact2: fourth-year-pms
+    damanMaster1: fourth-year-pms
```
Comment (JaspreetSanghra) [inline .travis.yml]: should this not be master now?

Comment (cmoihub) [reply .travis.yml]: I changed it

**Pull request merged**